lib/linguist/languages.yml

```diff
@@ -3458,6 +3458,14 @@ Pod:
   - perl
   tm_scope: none
   language_id: 288
+Pod6:
+  type: prose
+  ace_mode: perl6
```

Comment:
You should add this line to enable Perl/Pod 6 highlighting:

~~~yaml
+  tm_scope: source.perl6fe
~~~

Perl 6's grammar [includes support](https://github.com/perl6/atom-language-perl6/blob/master/grammars/perl6fe.cson#L19) for Pod6 markup, judging by the comments and pattern blocks.